src/jsdoc.ts

```diff
@@ -150,7 +171,11 @@ export function parse(comment: string): {tags: Tag[], warnings?: string[]}|null
         warnings.push(`@${tagName} annotations are redundant with TypeScript equivalents`);
         continue;  // Drop the tag so Closure won't process it.
       } else if (arrayIncludes(JSDOC_TAGS_WITH_TYPES, tagName) && text[0] === '{') {
-        warnings.push('type annotations (using {...}) are redundant with TypeScript types');
+        warnings.push(`the type annotation on @${tagName
+                      } is redundant with its TypeScript type, remove the {...} part`);
+                      continue;
```

Comment:
This is fixed in the newer `clang-format`, just approve that and I'll fixup the code here.